### ISSUE TITLE
Create POA request accept/decline flow

### ISSUE THREAD

**Issue by candilemoine** [department-of-veterans-affairs/va.gov-team#97664]
#### Mockups/Designs

- [Accept/decline flow](https://www.figma.com/design/LVCQBuW7a6nfVFNyhA4kv4/ARF---Form-21-22-Design-Explorations?node-id=1874-116166&node-type=section&t=rppQRSFyB6Y5B9HG-0)

### Acceptance Criteria/Conditions of Satisfaction

- [ ] 

#### Accessibility (A11y) Testing 
- Refer to existing [VA design system](https://design.va.gov/components/form/)
- Test with either [WAVE](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh?hl=en-US&utm_source=ext_sidebar) or [AXE](https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US&utm_source=ext_sidebar) extension
- As you manually test, look for the following:
   - color contrast errors
   - keyboard navigation
     - When navigating with a keyboard, content should flow from left to right
   - heading order 
     - Headings should be in order, so h1, then h2, h3. This is so the screen reader can let the user know the natural order of the elements in page. The WAVE tool has a headings tab that will display the order.
   - proper tag elements
     - Since we are working with forms, make sure components have the proper tags, i.e.:
 <label for="lastName"...>Last Name</label> <input id="lastName"... />
- test with a screen reader 
  - Voiceover is built in with mac, use NVDA for windows

**Comment by nihil2501** [comment]
@jquispe-oddball You can implement against a mocked out API for now

**Comment by jquispe-oddball** [comment]
@candilemoine I have a new figma from Jen that includes a toast alert and slightly updated design:
https://www.figma.com/design/LVCQBuW7a6nfVFNyhA4kv4/ARF---Form-21-22-Design-Explorations?node-id=3072-376413&t=0PKIp54sTvS6XGFP-0

**Comment by candilemoine** [comment]
@jquispe-oddball I've added the 2 new bullets based on the design but confirming if it should be moved over to the Eng Handoff.

**Comment by candilemoine** [comment]
@jquispe-oddball per [the decision](https://dsva.slack.com/archives/C05SUUM4GAW/p1734559177362929), we're removing the last decline option that has free text.

**Comment by candilemoine** [comment]
Sorry @jquispe-oddball! Half reverse. Can we keep "I decline for another reason"? That way we're not forcing reps to choose the defined decline options.

cc @nihil2501 @seipeljl @Kate-allbee

**Comment by jquispe-oddball** [comment]
Pr merged in, waiting to test on staging before moving ticket over the QA status
cc: @candilemoine @nihil2501

**Comment by Kate-allbee** [comment]
@jquispe-oddball The design looks good! Right now when I click submit decision it turns on the error bar on the left of the radios, is that happening because we don't have the processing state yet? 

<img width="400" alt="Image" src="https://github.com/user-attachments/assets/3eecf0e3-f0b2-403b-be48-3862ae1e631f" />

**Comment by jquispe-oddball** [comment]
@Kate-allbee yes, the processing state is a separate ticket! If everything else looks great, then this ticket can be closed.

**Comment by Kate-allbee** [comment]
Looks good! @jquispe-oddball
